recipe/meta.yaml

```diff
@@ -7,56 +7,63 @@ package:
 
 source:
   url: https://github.com/google/jax/archive/{{ name }}-v{{ version }}.tar.gz
-  sha256: 07a602e4e2e37fe46c17eeb6c25e80932f6abc5d1dde59e7868e7e3bb08f46c6
-  patches:
-    - 0001-Don-t-add-dashes-to-version.patch
+  sha256: 73ebc7868631cd9d520385557bbd7f08762d748a5a6a1bebef0f3b8d7ba748ef
 
 build:
   number: 0
-  skip: true  # [win or py<37]
-
+  # ppc64le and s390x are missing bazel. Win not supported by jaxlib.
+  skip: true  # [ppc64le or s390x or win or py<37]
 requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-    - numpy                                  # [build_platform != target_platform]
-    - scipy                                  # [build_platform != target_platform]
-    - bazel 4.1.0
-    - bazel-toolchain
-    - m2-patch  # [win]
-    - patch     # [not win]
+    - python
+    # https://github.com/google/jax/blob/jaxlib-v0.3.25/build/build.py#L197
+    - bazel >=5.1.1
+    - git
   host:
     - python
     - pip
-    - numpy
-    - scipy
     - wheel
```

Comment:
`setuptools` is missing in `host`, https://github.com/google/jax/blob/8788a9438f7c7e00ccc8d60f7cb320177e3d9f80/jaxlib/setup.py#L15

Reply:
Thank you! I've added that.